### PR TITLE
Add Sass support

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by akatsoulas** [comment]
Let's fix the CSS issues in the category and entities pages before adding this.

**Comment by comzeradd** [comment]
Rebased to the recent css fixes

**Comment by akatsoulas** [comment]
Can you do a quick rebase?

**Comment by comzeradd** [comment]
PR rebased

File: coss/settings.py [hunk 1]
```diff
@@ -139,6 +139,9 @@
     'compressor.filters.css_default.CssAbsoluteFilter',
     'compressor.filters.cssmin.rCSSMinFilter'
 ]
+COMPRESS_PRECOMPILERS = (
+    ('text/scss', 'sass {infile} {outfile}'),
```
Comment (akatsoulas) [inline coss/settings.py]: Shouldn't this be:

```python
COMPRESS_PRECOMPILERS = (
    ('text/x-scss', 'sass --scss {infile} {outfile}'),
)
```

**Comment by comzeradd** [comment]
I updated the PR. sass autodetects the right syntax parse, but let's be explicit.

**Pull request merged**